.changeset/brave-rats-guess.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
This should be broken down in 3 different changesets, cf warning note at the end of the How to section of [the corresponding wiki page](https://github.com/LedgerHQ/ledger-live/wiki/Changesets#how-to)

Also, all changes are `minor` and none is `patch`